src/pages/products/sgid/boundaries/county.astro

```diff
@@ -0,0 +1,123 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import DirectDownloads from '@components/data/DirectDownloads.astro';
+import Disclaimer from '@components/data/Disclaimer.astro';
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import ExternalLink from '@components/page/ExternalLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah County Boundaries', // The ArcGIS Online title Goes Here (Utah blah blah)
+  description: `This data set represents county boundaries. This dataset reflects the most current version of Utah county boundaries plus modifications made to correct known issues along the Davis-Weber and Duchesne-Uintah county boundaries. (20111108) and boundary agreement (certified 20120612) between Juab and Millard Counties. And an adjustment between Morgan and Summit Counties (4/3/2013). Minor adjustments were made to align with the Newest PLSS-GCDB layers from BLM (CadNSDIv2), any where from 10 to 50 feet all in non-populated areas (7/18/2014). Minor adjustment between Davis and Weber County (20201105)Data is current through Nov. 15, 2021; Population Figures from 2020 Census. Population Estimates are Null until we get a 2021 update.`,
```

Comment:
**polish**: This is too long. Let's go for 1-3 sentences.

Reply:
I've shortened this.